.pipelines/e2e.yml

```diff
@@ -16,26 +20,43 @@ jobs:
 - job: E2E
   timeoutInMinutes: 180
   pool:
-    name: ARO-CI
-    demands: go-1.17
+    name: 1es-aro-ci-pool
   steps:
   - template: ./templates/template-checkout.yml
 
+  - script: |
+      set -xe
+      sudo rpm -ivh https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
+      sudo dnf install -y openvpn make
+    displayName: Setup (Container)
+    target: container
+
   - template: ./templates/template-az-cli-login.yml
     parameters:
       azureDevOpsJSONSPN: $(aro-v4-e2e-devops-spn)
   - template: ./templates/template-push-images-to-acr.yml
     parameters:
       rpImageACR: $(RP_IMAGE_ACR)
-      buildCommand: publish-image-aro
+  - script: |
+      make extract-aro-docker
+    displayName: Extract ARO binaries from build
 
   - script: |
+      az account set -s $AZURE_SUBSCRIPTION_ID
+      SECRET_SA_ACCOUNT_NAME=e2earosecrets make secrets
+      . secrets/env
+
       export CI=true
       . ./hack/e2e/run-rp-and-e2e.sh
-
-      run_vpn
       deploy_e2e_db
+    displayName: Setup (Azure)
+
+  - script: |
+      export CI=true
+      . secrets/env
+      . ./hack/e2e/run-rp-and-e2e.sh
 
+      run_vpn
```

Comment:
I am probably missing something, but isn't that function removed?

Reply:
there was a duplicate in the file I removed :D